internal/controller/nopresource/nopresource.go

```diff
@@ -71,12 +71,14 @@ func reconcileLogic(conditionAfter []v1alpha1.ResourceConditionAfter, timeElapse
 		}
 	}
 
-	idx := make([]int, len(latestIdx))
+	idxs := make([]int, len(latestIdx))
```

Comment:
Ah sorry about that, thanks for catching! Intention was the following:
```suggestion
	idxs := make([]int, 0, len(latestIdx))
```
Which would set capacity to length and size to `0`, which would make just calling append work 👍